webapp/src/actions/createPullRequest.ts

```diff
@@ -71,16 +72,18 @@ export default async function sendPullRequest(
     }
 
     const nowIso = new Date().toISOString().replace(/:/g, '').split('.')[0];
-    const branchName = 'lyra-translate-' + nowIso;
+    const uuidSnippet = randomUUID().substring(5);
+    const branchName = 'lyra-translate-' + uuidSnippet;
+
     await repoGit.newBranchCommitAndPush(
       branchName,
       langFilePaths,
-      `Lyra translate: ${nowIso}`,
+      `Lyra translate: ${nowIso}-${uuidSnippet}`,
     );
 
     const pullRequestUrl = await repoGit.createPR(
       branchName,
-      'LYRA Translate PR: ' + nowIso,
+      'LYRA Translate PR: ' + uuidSnippet,
```

Comment:
For pull request titles, nowIso will be more helpful than `uuidSnippet`. A developer needing to distinguish two pull requests, already has access to the branch name.

Reply:
I understand and agree, I will change this back. :)